scripts/pre_commit_linter.py

```diff
@@ -1183,6 +1183,15 @@ def _check_docstrings(all_files):
                         is_docstring = True
                         is_class_or_function = False
 
+                # Check for space after """ in docstring.
+                if re.match(r'^""".+$', line) and is_docstring and (
+                        line[3] == ' '):
+                    failed = True
+                    print (
+                        '%s --> Line %s: There should be no space after """ '
```

Comment:
Pull this message out and put it below existing message variables.

Reply:
done